string environment_id
HumanNaviObjectInfo[] objects_info
HumanNaviObjectInfo target_object
geometry_msgs/Point destination
